src/pages/home/HeaderView.js

```diff
@@ -93,10 +93,15 @@ function HeaderView(props) {
     const {translate} = useLocalize();
     const theme = useTheme();
     const styles = useThemeStyles();
-    const participants = lodashGet(props.report, 'participantAccountIDs', []);
+    const isSelfDM = ReportUtils.isSelfDM(props.report);
+    let participants = lodashGet(props.report, 'participantAccountIDs', []);
+    if (isSelfDM) {
+        participants = [props.session.accountID];
+    }
```

Comment:
NAB: This will allow `participants` to remain `const`:
```suggestion
    const participants = isSelfDM ? [props.session.accountID] : lodashGet(props.report, 'participantAccountIDs', []);
```